### DISCUSSION TITLE
Thick Cursor in new kitty terminal

### DISCUSSION THREAD

**Discussion by ciscohack** [kovidgoyal/kitty#6852] [General]
In the latest Kitty terminal, the cursor changed to a thick cursor automatically.  Although I have a setting for a  thin cursor. Is there a way to prevent this transition?

here is my normal cursor SS
<img width="110" alt="image" src="https://github.com/kovidgoyal/kitty/assets/11479557/b739a86c-594a-4729-8510-d3c698c50cef">

and now cursor changed to thick  

<img width="104" alt="image" src="https://github.com/kovidgoyal/kitty/assets/11479557/83dbeebf-ca4f-4f20-b672-8e28dfa3a40f">

**Comment by Luflosi** [comment]
Try setting `cursor_shape beam`. See https://sw.kovidgoyal.net/kitty/conf/#opt-kitty.cursor_shape.

**Comment by ciscohack** [comment]
@Luflosi  Thanks for the comment. I already have the config here is the config snippet still having problem started recently only earlier it was working fine

cursor_shape beam
cursor_beam_thickness 4
cursor_underline_thickness 4.0
